wiki/views/admin_panel.py

```diff
@@ -0,0 +1,25 @@
+from django.views.generic.base import TemplateView
+from django.shortcuts import redirect
+
+from wiki import models
+
+class AdminPanelView(TemplateView):
+
+    template_name = "wiki/admin_panel.html"
+
+    def dispatch(self, request, *args, **kwargs):
+        # Let logged in super users continue
+        if not request.user.is_superuser:
+            return redirect('wiki:root')
+
+        return super(AdminPanelView, self).dispatch(request, *args, **kwargs)
+
+    def get_context_data(self, **kwargs):
+        articleList = models.Article.objects.all()
```

Comment:
Could you do `article_list` and `deleted_articles` instead of `articleList` and `deletedArticles` here? The camelCase convention isn't so pythonic :)